src/codemagic/tools/_app_store_connect/arguments.py

```diff
@@ -889,6 +901,32 @@ class PublishArgument(cli.Argument):
             'required': False,
         },
     )
+    EXPIRE_PREVIOUS_BUILDS = cli.ArgumentProperties(
+        key='expire_previous_builds',
+        flags=('--expire-previous-builds', '-eb'),
```

Comment:
Try to avoid two-letter alternatives for long flags. In case it is not possible with one-letter, then perhaps it is better to omit the shorthand version altogether.